code/modules/mob/living/carbon/species.dm

```diff
@@ -611,6 +640,29 @@
 
 	reagent_tag = IS_DIONA
 
+	has_organ = list(
+		"nutrient channel" =   /obj/item/organ/diona/nutrients,
+		"neural strata" =      /obj/item/organ/diona/strata,
+		"response node" =      /obj/item/organ/diona/node,
+		"gas bladder" =        /obj/item/organ/diona/bladder,
+		"polyp segment" =      /obj/item/organ/diona/polyp,
+		"anchoring ligament" = /obj/item/organ/diona/ligament
+		)
+
```

Comment:
News to me that a fully featured system for arbitrary organs is 'snowflake'. :P If this was a full port, the purpose of the filler organs was to make dionaea totally incompatible with transplants (and also make them explode into nymphs when gibbed).